Inc/usbplayback/inputs.h

```diff
@@ -0,0 +1,30 @@
+#ifndef INPUTS_H_
+#define INPUTS_H_
+
+#define DEBOUNCETIME 1
+
+// How long a button has to be pressed before it's considered held
```

Comment:
Could you please add to the comments what this unit of time is? Cycles of the main loop?

Reply:
It's supposed to be in milliseconds but because the main loop also does the OLED it ends up being a minimum of about 25ms no matter what you set. I will decouple them by running the input processing in a low-priority interrupt, I think that makes more sense.

Reply:
OK input processing runs in TIM2 now, USB and screen still run in the main loop as they're buffered so not timing-sensitive.